install.sh

```diff
@@ -268,6 +268,16 @@ rosdep install -r --from-paths src --ignore-src # Be careful not to install libo
 sudo apt purge ros-melodic-libg2o
 catkin_make
 
+# udev rules of opencm setting
+echo -e "\e[33m#####################################"
+echo -e "\e[33m# udev rules for opencm             #"
+echo -e "\e[33m#####################################"
+
+echo "ATTRS{idVendor}==\"fff1\" ATTRS{idProduct}==\"ff48\", ENV{ID_MM_DEVICE_IGNORE}=\"1\", MODE:=\"0666\", SYMLINK+=\"ultrasonics\"" >> 99-opencm-cdc.rules
+sudo cp 99-opencm-cdc.rules /etc/udev/rules.d/
```

Comment:
I like `mv` than `cp` because I don't hope git diff remains in this repository after installing...

Reply:
yea, that's better I will change